docs/tutorials/ctapipe_handson.ipynb

```diff
@@ -0,0 +1,659 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting Started with ctapipe\n",
+    "\n",
+    "This hands-on was presented at the Paris CTA Consoritum meeting (K. Kosack)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part 1: load and loop over data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ctapipe.io import event_source\n",
+    "from ctapipe import utils\n",
+    "from matplotlib import pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = utils.get_dataset_path(\"gamma_test.simtel.gz\")\n",
```

Comment:
I think the return value of `get_dataset_path` is a path-like or string pointing to a file containing *data* .. it is not the *data* itself. Therefore I think this should be named `path`